osquery/tables/networking/posix/curl.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <boost/network/include/http/client.hpp>
+#include <boost/numeric/conversion/cast.hpp>
+#include <chrono>
+#include <osquery/tables.h>
+
+#include <stdio.h>
+
```

Comment:
Nit: we generally do imports in the order of std, then third-party, then osquery.

e.g.
```
#include <chrono>
#include <stdio.h>

#include <boost/network/include/http/client.hpp>
#include <boost/numeric/conversion/cast.hpp>

#include <osquery/tables.h>
```